### DISCUSSION TITLE
Teleport no longer works in Laravel 11 after optimize command

### DISCUSSION THREAD

**Discussion by mrfelipemartins** [livewire/livewire#8130] [7. Bugs (archived)]
### Livewire version

v3.4.9

### Laravel version

v11.0.6

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Create a new Livewire Component
2. Add a @teleport directive
3. Run` php artisan optimize` command
4. Check your component

### Repository or Wirebox URL

https://github.com/mrfelipemartins/tests-laravel-11

### Additional information

Teleporting works until optimize command is called, then we see it as plain text.

Before optimize:
<img width="639" alt="image" src="https://github.com/livewire/livewire/assets/24225909/9dca379b-6ad8-4723-a1da-c4d0a68da604">

After optimize:
<img width="173" alt="image" src="https://github.com/livewire/livewire/assets/24225909/70bad627-00d9-42cf-b9aa-316253c5f125">

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@mrfelipemartins thanks for reporting. Do you know if it also happens on Laravel 10 or whether its an issue specific to Laravel 11?

**Reply by mrfelipemartins** [reply]
I would have to double check but I am pretty sure it's Laravel 11 specific since I noticed this problem after migrating my project from 10 to 11

**Reply by joshhanley** [reply]
@mrfelipemartins if you get a chance, can you check 10? As that will help us work out whether it's a Laravel 11 change that we need to track down or whether the feature just has a bug in general. Thanks!

**Reply by mrfelipemartins** [reply]
@joshhanley I can confirm it's only happening in Laravel 11. Just ran a test in a fresh 10 and it worked as expected.

**Reply by edvordo** [reply]
Can also confirm this is only happening with laravel 11 when I run `view:cache`. 

https://github.com/livewire/livewire/blob/cac2f7b28f37980653aaec99d9de55de21b2ebaa/src/Features/SupportTeleporting/SupportTeleporting.php#L11-L13

I added some debug output and discovered the above directive callback is ran at run time, but never with `view:cache`.

**Reply by luanfreitasdev** [reply]
I can confirm that this also happens with Laravel 10

**Reply by luanfreitasdev** [reply]
https://github.com/livewire/livewire/pull/8282
